lib/cli/src/Cardano/CLI.hs

```diff
@@ -501,9 +594,33 @@ cmdRootKey =
         scheme :: CliKeyScheme Text IO
         scheme =
             hoistKeyScheme eitherToIO
-            . mapKey xPrvToHexText
+            . mapKey xPrvToTextTransform
             $ newCliKeyScheme keyType
 
+data KeyChildArgs = KeyChildArgs
+    { _path :: [DerivationIndex]
+    , _walletStyle :: CliWalletStyle
+    , _key :: Text
+    }
+
+cmdChildKey :: Mod CommandFields (IO ())
+cmdChildKey =
+    command "child" $ info (helper <*> cmd) $ mempty
+        <> progDesc "Derive child keys."
+  where
+    cmd = fmap exec $
+        KeyChildArgs <$> pathOption <*> walletStyleOption <*> keyArgument
+
+    exec (KeyChildArgs path style k0) = do
+        child <- foldM (deriveChildKey scheme) k0 path
+        TIO.putStrLn child
+      where
+        scheme :: CliKeyScheme Text IO
+        scheme =
+            hoistKeyScheme eitherToIO
+            . mapKey xPrvToTextTransform
+            $ newCliKeyScheme style
```

Comment:
Ok, I see 

> No, rather a --byron flag, if present, use derivation scheme 1, if not, use derivation scheme 2.

that we shouldn't take `--wallet-style` as an argument.

🤔

Reply:
I guess one _could_ strategically use a hard-coded `Icarus` scheme here and leave a todo for potential second U/S. And add three goldens for `key child` such that they don't end up diverging by accident.

Reply:
>  I guess one could strategically use a hard-coded `Icarus`

Done.